runtime-light/stdlib/job-worker/job-worker-api.cpp

```diff
@@ -0,0 +1,153 @@
+}
+
+} // namespace
+
+// ================================================================================================
+
+task_t<Optional<int64_t>> f$kphp_job_worker_start(string request, double timeout) noexcept {
+  const auto fork_id{co_await kphp_job_worker_start_impl(std::move(request), timeout, false)};
+  co_return fork_id != INVALID_FORK_ID ? fork_id : false;
+}
+
+task_t<bool> f$kphp_job_worker_start_no_reply(string request, double timeout) noexcept {
+  const auto fork_id{co_await kphp_job_worker_start_impl(std::move(request), timeout, true)};
+  co_return fork_id != INVALID_FORK_ID;
+}
+
+task_t<array<Optional<int64_t>>> f$kphp_job_worker_start_multi(array<string> requests, double timeout) noexcept {
+  array<Optional<int64_t>> fork_ids{requests.size()};
+  for (const auto &it : requests) {
+    const auto fork_id{co_await kphp_job_worker_start_impl(it.get_value(), timeout, false)};
+    fork_ids.set_value(it.get_key(), fork_id != INVALID_FORK_ID ? fork_id : false);
+  }
+  co_return fork_ids;
+}
+
+// ================================================================================================
+
+task_t<string> f$kphp_job_worker_fetch_request() noexcept {
+  if (!f$is_kphp_job_workers_enabled()) {
+    php_warning("couldn't fetch job worker request: job workers are disabled");
+    co_return string{};
+  }
+
+  auto &jw_server_ctx{JobWorkerServerComponentContext::get()};
+  if (jw_server_ctx.job_id == JOB_WORKER_INVALID_JOB_ID || jw_server_ctx.body.empty()) {
+    php_warning("couldn't fetch job worker request");
+    co_return string{};
+  }
+  co_return std::move(jw_server_ctx.body);
+}
+
+task_t<int64_t> f$kphp_job_worker_store_response(string response) noexcept {
+  auto &component_ctx{*get_component_context()};
+  auto &jw_server_ctx{JobWorkerServerComponentContext::get()};
+  if (!f$is_kphp_job_workers_enabled()) { // workers are enabled
+    php_warning("couldn't store job worker response: job workers are disabled");
+    co_return static_cast<int64_t>(JobWorkerError::store_response_incorrect_call_error);
+  } else if (jw_server_ctx.kind != JobWorkerServerComponentContext::Kind::Regular) { // we're in regular worker
+    php_warning("couldn't store job worker response: we are either in no reply job worker or not in a job worker at all");
+    co_return static_cast<int64_t>(JobWorkerError::store_response_incorrect_call_error);
+  } else if (jw_server_ctx.state == JobWorkerServerComponentContext::State::Replied) { // it's the first attempt to reply
+    php_warning("couldn't store job worker response: multiple stores are forbidden");
+    co_return static_cast<int64_t>(JobWorkerError::store_response_incorrect_call_error);
+  } else if (component_ctx.standard_stream() == INVALID_PLATFORM_DESCRIPTOR) { // we have a stream to write into
+    php_warning("couldn't store job worker response: no standard stream");
```

Comment:
I think message `no standard stream` maybe quite unexpected for php developer. Maybe add here info that connection closed?

Reply:
We don't know the reason why `standard_stream` is unset, so I decided that such a general message is enough. At least for now.